### PR TITLE
Task 13948140: Update C# SDK repository to consume new api version

### PR DESCRIPTION
**Pull request by thomasp98296**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: src/SDKs/_metadata/monitor_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+2022-05-23 16:47:10 UTC
```
Comment (Yao725) [inline src/SDKs/_metadata/monitor_resource-manager.txt:1]: Where does this file come from? I couldn't understand why we need it.

File: sdk/monitor/Microsoft.Azure.Management.Monitor/src/Generated/ActionGroupsOperations.cs [hunk 2]
```diff
@@ -920,10 +920,57 @@ public async Task<AzureOperationResponse<ActionGroupResource>> UpdateWithHttpMes
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse<TestNotificationResponse>> PostTestNotificationsWithHttpMessagesAsync(NotificationRequestBody notificationRequest, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationHeaderResponse<ActionGroupsPostTestNotificationsHeaders>> PostTestNotificationsWithHttpMessagesAsync(NotificationRequestBody notificationRequest, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (Yao725) [inline sdk/monitor/Microsoft.Azure.Management.Monitor/src/Generated/ActionGroupsOperations.cs:923]: I don't see any change in test, do these updated APIs have corresponding tests? If not, could you add tests to see if these APIs could work as expected.

**Review by Yao725 — CHANGES_REQUESTED** [review]
You also need to update following contents:

- `Version` and `PackageReleaseNotes` in Microsoft.Azure.Management.Monitor.csproj
- `AssemblyVersion` and `AssemblyFileVersion` in AssemblyInfo.cs
- `AzureApiTag` in AzSdk.RP.props

It looks like your SDK is still in preview status, If so, you need to bump the minor version. If not, you need to bump the major version because of breaking changes.

**Comment by thomasp98296** [comment]
@Yao725 , thanks for your feedback. I have sent another iteration. Please help to take another look. Thank you again.

**Review by Yao725 — APPROVED** [review]
LGTM, and FYI @thomasp98296, I helped you bump the version in the csproj.

**Pull request merged**